packages/DataHeader/stories/DataHeader.stories.mdx

```diff
@@ -0,0 +1,154 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { getStoryName } from "storybook/storyTree";
+import Readme from "../README.md";
+import { Basic, WithRenderActions, WithTypes, WithBackgroundColor } from "./DataHeader.examples";
+import Toast from "@paprika/toast";
+
+export const params = {
+  options: {
+    isToolshown: true,
+    showPanel: false,
+  },
+};
+
+<Meta title={`${getStoryName("DataHeader")}/Docs`} parameters={params} />
+
+# DataHeader
+
+## Introduction
+
+The `<DataHeader />` component helps consumer to deal with header with the `<Table />` and `<DataGrid />` components.
+
+With this component you have access to three major features: render Label via a string or function, render an action button/menu at the right side of the header or render an icon `type`
+before the label.
+
+The component looks like:
+
+```js
+<DataHeader label={string|function} type={:string} renderActions={:function} />
+```
+
+## Basic example
+
+A basic example using a `<Table />`
+
+```js
+<Table data={data}>
+  <Table.ColumnDefinition header={() => <DataHeader label="name" />} cell="name" />
+</Table>
+```
+
+### Output
+
+<Basic />
+
+## Render Actions Prop
+
+The previous example is too simple, you can achieve the same using a single `header` prop on the `ColumnDefinition`
+Let's add the `renderActions` prop to make it more interesting:
+
+```js
```

Comment:
`jsx` produces nicer syntax highlighting, in my opinion.

Reply:
what's the difference? adding those ticks is quite simply no need to learn anything neither remember :)

Reply:
Just a suggestion from what I've found.  Using `'''jsx` instead of `'''js` makes it prettier.

![Screen Shot 2020-10-15 at 10 24 53 AM](https://user-images.githubusercontent.com/14944896/96165220-33815600-0ed1-11eb-9b87-2b1210679ccc.png)

vs

![Screen Shot 2020-10-15 at 10 25 13 AM](https://user-images.githubusercontent.com/14944896/96165227-354b1980-0ed1-11eb-84bb-735c158a01c7.png)